src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/Subscriptions/DependenciesSnapshotProvider.cs

```diff
@@ -294,6 +294,8 @@ private void DisposeCore()
             ITargetFramework? activeTargetFramework,
             CancellationToken token)
         {
+            Assumes.NotNull(_commonServices.Project.FullPath);
```

Comment:
This path will be null if the project is not loaded from disk. Our projects are always loaded from disk.